lib/transports.js

```diff
@@ -6,12 +6,16 @@ var util = require('util');
 function Transport() {}
 util.inherits(Transport, events.EventEmitter);
 
+var agentOptions = { keepalive: true, maxSockets: 100 };
+
 var http = require('http');
+var httpAgent = new http.Agent(agentOptions);
 
 function HTTPTransport(options) {
   this.defaultPort = 80;
   this.transport = http;
   this.options = options || {};
+  this.agent = httpAgent;
```

Comment:
We could theoretically enable users to change agent options via something like:
```javascript
    Raven.config('___DSN___', {
      transport: new raven.transports.HTTPSTransport({agentOpts: { ... }})
    });
```
but I'm not sure whether we need to or should; are there cases where users might want to do this (or we might want them to)? Guessing you have more background on our intentions behind transports being pluggable/user-configurable.

Reply:
Agree on enabling user to change those settings.

Reply:
+ adding a comment in the docs about it